packages/adblocker-electron/src/index.ts

```diff
@@ -164,101 +158,65 @@ export class ElectronBlocker extends FiltersEngine {
   // ElectronBlocker-specific additions to FiltersEngine
   // ----------------------------------------------------------------------- //
 
-  public onIsMutationObserverEnabled = (event: Electron.IpcMainEvent): void => {
-    event.returnValue = this.config.enableMutationObserver;
+  public onIsMutationObserverEnabled = async (_: Electron.IpcMainInvokeEvent): Promise<boolean> => {
+    return this.config.enableMutationObserver;
   };
```

Comment:
Does this need to be `async`?

Reply:
The function needs to return a `Promise`, and the `async` keyword wraps the body in a promise automatically here.

Reply:
Makes sense. Thanks.